.circleci/config.yml

```diff
@@ -284,65 +300,76 @@ jobs:
       - setup_vm
       - install_chrome
       - run:
-          name: 'Module Tests (<< parameters.config >>)'
+          name: '⭐ Module Tests (<< parameters.config >>) ⭐'
           command: node build-system/pr-check/module-tests.js --config=<< parameters.config >>
       - store_test_output
       - teardown_vm
   'End-to-End Tests':
     executor:
-      name: amphtml-medium-executor
+      name: docker-medium
     parallelism: 6
     steps:
       - setup_vm
       - install_chrome
       - run:
-          name: 'End-to-End Tests'
+          name: '⭐ End-to-End Tests ⭐'
           command: node build-system/pr-check/e2e-tests.js
       - store_test_output
       - teardown_vm
+  'Safari Tests':
```

Comment:
How about `Browser Tests (Safari)` - this way when we add Edge, Firefox, whatever they would be `Browser Tests (Edge)` etc

Reply:
Thought of doing something this. Initially tried `Cross Browser Tests (Safari)`, the same name that's being migrated from GH Actions, but the operative part would get truncated in the mobile view.

Gonna punt on this for now and standardize the job nomenclature in the next PR that adds the other browsers. SGTY?

Reply:
SGTM